.github/workflows/build-test.yml

```
@@ -366,6 +366,27 @@ jobs:
           make --touch codegen schema
           make provider_prebuild
 
+      - name: Build and sign windows provider
+        run: |
```

Shouldn't we use `set -euxo pipefail` or similar here to detect issues and fail early?